### PR TITLE
add lint and package-lock in clean

### PR DESCRIPTION
**Pull request by MichelML**

### CODE REVIEW

File: gulpTasks/svg.js [hunk 1]
```diff
@@ -1,68 +1,70 @@
...
+            code += '        ' + camelizedKey + ': { name : \'' + camelizedKey + '\', render : function(svgClass, spanClass, title, attr) { return svgWrapper(' + JSON.stringify(that.json[key]) + ', svgClass, spanClass, title, attr); } }, \n';
         });
         code += '    };';
 
         fs.writeFileSync(to, code);
     };
 }
 
-gulp.task('svg:concat', 'Concat all svg files into one in a json format and export it to dist/svg', function () {
+gulp.task('svg:concat', 'Concat all svg files into one in a json format and export it to dist/svg', function() {
```
Comment (sroy3) [inline gulpTasks/svg.js]: Arrow function

File: gulpTasks/svg.js [hunk 2]
```diff
@@ -1,68 +1,70 @@
...
-            }]
+                removeComments: true,
+            }],
         }))
-        .pipe(cheerio(function ($) {
-            $('svg').each(function () {
-                var svg = $(this);
+        // tslint:disable
+        .pipe(cheerio(function($) {
+            $('svg').each(function() {
```
Comment (sroy3) [inline gulpTasks/svg.js]: Arrow functions

File: gulpfile.js [hunk 3]
```diff
@@ -2,20 +2,20 @@
 // --min                        Flag to enable minification                                         Default: false
 // --gzip                       Flag to enable gziphication                                         Default: false
 
-var gulp = require('gulp-help')(require('gulp'));
-var requireDir = require('require-dir');
-var tasks = requireDir('./gulpTasks');
+let gulp = require('gulp-help')(require('gulp'));
+let requireDir = require('require-dir');
+let tasks = requireDir('./gulpTasks');
```
Comment (sroy3) [inline gulpfile.js]: const

File: gulpTasks/svg.js (hunk 2 above)
Comment (MichelML) [reply gulpTasks/svg.js]: on a besoin que ca soit la pour le `this`

File: gulpTasks/svg.js [hunk 4]
```diff
@@ -1,53 +1,54 @@
...
 
-    this.writeSvgEnumFile = function (to) {
-        var code = 'var svgEnum = {\n';
-        var that = this;
-        _.each(_.keys(this.json), function (key) {
-            var camelizedKey = s.camelize(key);
-            code += '        ' + camelizedKey + ': { name : \'' + camelizedKey + '\', render : function(svgClass, spanClass, title, attr) { return svgWrapper(' + JSON.stringify(that.json[key]) + ', svgClass, spanClass, title, attr); } }, \n'
+    this.writeSvgEnumFile = (to) => {
+        let code = 'var svgEnum = {\n';
+        const that = this;
```
Comment (GermainBergeron) [inline gulpTasks/svg.js]: You can replace `that` with `this` since you use an arrow function

**Pull request merged**